common/src/main/kotlin/flank.common/Files.kt

```diff
@@ -0,0 +1,50 @@
+package flank.common
+
+import com.github.kittinunf.fuel.Fuel
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+
+val userHome: String by lazy {
+    if (isWindows) System.getenv("HOMEPATH") else System.getProperty("user.home")
```

Comment:
Need me to test locally on my machine?

Reply:
I just moved this code, should work properly 🤞